components/chapterList.js

```diff
@@ -65,15 +65,23 @@ export default class ChapterList extends React.Component {
   }
 }
 
-class Chapter extends React.PureComponent {
+@observer
```

Comment:
still not sure why `.slice()` on the array _didn't_ work, but making this an observer _did_ work... that's the opposite of how it worked for the Manga LIst...

need to get better at understanding MobX gotchas...